app/controllers/api/v0x1.rb

```diff
@@ -1,5 +1,11 @@
 module Api
   module V0x1
+    class RootController < ApplicationController
+      def openapi
+        render :json => File.read(Rails.root.join('public', 'catalog', 'v0.1.0', 'openapi.json'))
```

Comment:
Why are there 3 digit routes?  we only use 2 digit.

Reply:
This is fine for now as it's under the covers, but we should change this later.

Also, minor but this can be `Rails.root.join('public', 'catalog', 'v0.1.0', 'openapi.json').read`

Reply:
@syncrou Will you apply these changes to your v1.0 updates?